src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PossibleDeNovo.java

```diff
@@ -39,11 +40,24 @@ public final class PossibleDeNovo extends PedigreeAnnotation implements InfoFiel
     private final MendelianViolation mendelianViolation;
     private Set<Trio> trios;
 
+    @Argument(fullName = "denovo-parent-gq-threshold", doc = "Minimum genotype quality for parents to be considered for de novo calculation (separate from GQ thershold for full trio).", optional = true)
```

Comment:
Now that we have these new arguments can you add a test into the GATKAnnotationPluginDescriptorUnitTest to be really certain that this machinery works reasonably well with the pedigree file + these extra arguments? Sorry to ask this of you but we don't have a lot of examples of annotation arguments and I want to be sure it really works in the engine as expected.

Reply:
I tried to add this test, but please take a look and make sure it's doing what you expected.